src/index.js

```diff
@@ -85,6 +105,23 @@ function taskListDisplayed() {
   });
 }
 
+function clearAllCompletedTasks() {
+  if (dataStorage !== null) {
+    toDoTasks = dataStorage;
+  }
+  const completedTasks = [];
+  toDoTasks.forEach((task) => {
+    if (task.completed) {
+      completedTasks.push(task);
+    }
+  });
+  completedTasks.forEach((task) => {
+    Crud.removeTask(toDoTasks, task.index);
+  });
+  setLocalStorage();
+  taskListDisplayed();
+}
```

Comment:
- [ ] You should implement a function for the "Clear all completed" button using filter() method as it is the project's requirement. It is more efficient. :pray:

Reply:
**Thank's @henatan99  for the review, I have solved it! 😄**